aliyun_img_utils/aliyun_cli.py

```diff
@@ -511,6 +511,68 @@ def publish(context, image_name, launch_permission, regions, **kwargs):
         )
 
 
+@click.command()
+@click.option(
+    '--image-name',
+    type=click.STRING,
+    help='Name of the image to be unpublished.',
+    required=True
+)
+@click.option(
+    '--launch-permission',
+    type=click.STRING,
+    help='The launch permission to set for the unpublished image.',
```

Comment:
Hmm, doesn't the description "unpublished image" imply the launch permissions to be "PRIVATE"?

Or is there a mode in aliyun where an image wold be "unpublished" but the "launch-permission" could be set up for certain accounts? In which case we probably want a different description.